src/Dev/Tasks/TagsToShortcodeHelper.php

```diff
@@ -313,20 +314,18 @@ private function getNewTag($tag)
                     '/href\s*=\s*(?:"|\').*?(?:"|\')/i',
                     '/id\s*=\s*(?:"|\').*?(?:"|\')/i',
                     '/\s*(\/?>|\])/',
-                    '/\s?\S+=("")|(\'\')/'
```

Comment:
This line was meant to remove empty attributes. e.g.: `alt=""` would have been stripped from the converted short code.

I don't think this is necessary. Many empty attribute carry meaning ... e.g. a blank `alt` will squash accessibility warning.

@bergice Do you remember if there was any specific use case you had in mind for this?

Reply:
@chillu wanted to keep this so that we don't change the structure of the content more than we need to.

I'm okay with removing it though.

Reply:
Blank alt tags have semantic meaning (accessibility), so we shouldn't remove them unless it's a lot of work to keep them. But I don't feel strongly about it either way.